src/sentry/workflow_engine/migrations/0052_add_legacy_rule_indices.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.1.7 on 2025-04-30 23:08
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # Run manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+    atomic = False
+
+    is_post_deployment = True
+
+    dependencies = [
+        ("workflow_engine", "0051_migrate_remaining_issue_alerts"),
+    ]
+
+    operations = [
+        # Add index to rule_id - this column already has values
+        migrations.RunSQL(
+            "CREATE INDEX CONCURRENTLY idx_alertruleworkflow_rule_id ON workflow_engine_alertruleworkflow (rule_id) WHERE rule_id IS NOT NULL;",
+            "DROP INDEX CONCURRENTLY IF EXISTS idx_alertruleworkflow_rule_id;",
+        ),
+        # Add index to alert_rule_id
+        migrations.RunSQL(
+            "CREATE INDEX CONCURRENTLY idx_alertruleworkflow_alert_rule_id ON workflow_engine_alertruleworkflow (alert_rule_id) WHERE alert_rule_id IS NOT NULL;",
+            "DROP INDEX CONCURRENTLY IF EXISTS idx_alertruleworkflow_alert_rule_id;",
+        ),
+    ]
```

Comment:
You should just add these indexes to the Django model and it'll generated this for you. We prefer to avoid custom sql whenever possible

Reply:
are there any docs on how to add it with `CREATE INDEX CONCURRENTLY` (i wasn't sure based on [the migration documentation](https://develop.sentry.dev/backend/application-domains/database-migrations/#indexes))

Reply:
The docs say
> Our migration framework will do this automatically when creating a new index

Maybe we just need to reword it to make it less verbose?

Reply:
yeah, i think that'd help. i mostly keyed in on the syntax highlighting and saw the create index, and atomic bits.